packages/data-service-generator/src/admin/create-admin.ts

```diff
@@ -39,10 +39,10 @@ export function createAdminModules(): Promise<ModuleMap> {
 
 async function createAdminModulesInternal(): Promise<ModuleMap> {
   const context = DsgContext.getInstance;
```

Comment:
Thanks @xonx4l for the contribution.
Anyway, there are more places in this and other files where we need a bit of cleanup.
My suggestion is to remove this line here (and the other files) and start fixing the code on the go.
```suggestion
```